sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1309,6 +1309,13 @@ object CodeGenerator extends Logging {
   // bytecode instruction
   final val MUTABLESTATEARRAY_SIZE_LIMIT = 32768
 
+  // Display the total compile time for generating java code in the output of the test case.
+  private val _compileTime = new LongAccumulator
```

Comment:
can we add a simple test to run queries with whole-stage-codegen on/off and see if this metrics increase? whole-stage-codegen compiles code at driver side.

Reply:
OK. I added the test data onto the description of this PR.